client/src/assets/FormModal.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import StyleLib from './Stylesheet.jsx';
+import StarsInput from './StarsInput.jsx';
+import styled from 'styled-components';
+import Modal from 'react-modal';
```

Comment:
I don't think this modal import adds too much except basic form, so I don't think it's a big deal, but I feel like I remember somewhere that someone told us to build the modal from scratch. More awareness than anything, because I couldn't find a written requirement.